openfisca_france/parameters/chomage/allocations_assurance_chomage/alloc_base/montant_minimum/3_premiers_mois.yaml

```diff
@@ -307,5 +240,5 @@ metadata:
     - title: Protocole d'accord du 18/07/1992 et Convention du 01/01/1993 créent l'allocation unique dégressive
     2001-07-01:
     - title: "Nouvelle convention: création de l'Allocation de retour à l'emploi qui remplace AUD"
-    2004-07-01:
-    - title: Valeur inconnue de l'allocation journalière versée pendant une période de formation.
+    2002-07-01:
+    - title: La séparation de temps n'existe plus, voir parameters.chomage.allocation_retour_emploi.montant_minimum_hors_mayotte
```

Comment:
@sylvainipp merci pour ces précisions

Même si je comprends bien l'objectif, je pense qu'il faudrait éviter de mentionner le chemin du nouveau paramètre sous forme de notation "pointée" mais juste dire voir le montant minimum hors mayotte de l'ARE.
Notamment parce que ces éléments vont apparaître dans des interfaces qui vont les afficher comme le site des barèmes IPP.

Par contre, rajouter une explication plus longue en documentation pourrait être utile.
Si tu as le temps, pourrais-tu jeter un œil au résultat sur une [MR mirroir de celle-ci](https://gitlab.com/ipp/partage-public-ipp/baremes-ipp/baremes-ipp-yaml/-/merge_requests/449) ?
Je te propose d'y modifier ce qui est nécessaire pour retourner un rendu correct sur le site des barèmes résultant qui est accessible en cliquant sur le lien "View app". Et je repropagerai toutes nos modifications sur cette PR in fine.

Est-ce que cela te convient @sylvainipp ?

Reply:
Cela me convient, je commence juste par gérer le problème du à l'appel de l'ancien paramètre.

Reply:
Après réflexion, je propose dans les barèmes de modifier l'emplacement du fichier, donc il faudra dans tous les cas repasser sur tous les liens si tu penses que la modification est bonne (je laisse donc des erreurs).